### ISSUE TITLE
Documentation Task 1: Installation Troubleshooting

### ISSUE THREAD

**Issue by Theodlz** [grandma-collaboration/skyportal_grandma_dev#14]
As mentioned by Michael during last week's meeting, it would be amazing if you guys could create a document where you list the issues you encountered and the solutions.

It should be only one document, where all of you contribute by sharing your difficulties and how you tackled them.

The sooner the better, as this will be extremely helpful for those of you that didn't manage to run the app yet, so please make this a priority if you encountered a bunch of issues (so especially those that don't have the app running yet should focus on that documentation).

Here's how you should proceed:

Start by creating a google doc where you can all very easily write everything down in a collaborative manner. Once you are done with the first iteration of the documentation, tag me under this issue.
Ideally, this should be ready fairly quickly, as it's mostly copying and pasting what you each have in your respective issues, with added details and comments.

I expect that you guys can do this in just a few days, so let's set a deadline for next Thursday (Nov. 10, 2022). 
If possible, let's try to all have the app running and the tutorial completed for the same date. 
The project started already a few weeks ago, so let's try to not take more than a month to get ready to start.

For the tutorial, it can be a week after as some of you don't have the app ready yet.

Please tag me under that issue if you need any help/comments, and tag me once more to notify me when a first draft of that document is ready.

Have a nice day and good luck! :rocket:

**Comment by Theodlz** [comment]
@KodeurFou
@Nayru12
@aolrc-vinci
@ManonFlt
@celiune
@laurall974
@Carla802
@jolyanem
@lydie10
@Adiman007

**Comment by Carla802** [comment]
Hi @Theodlz @mcoughlin,

This is the link to our google doc about the installation tutorial. 🚀 

https://docs.google.com/document/d/1OW1D3mPNSI1XxklTrBVk3u4ln_G19oePGlNcKEFq0OI/edit?usp=sharing

**Comment by mcoughlin** [comment]
@Carla802 @Theodlz maybe this could be turned into a FAQ portion of the install guide?

**Comment by Theodlz** [comment]
Definitely. You guys can re write this cleanly in markdown format. That will be a great opportunity to learn how to compile the docs as well. It's all detailed in the documentation :)

**Comment by Theodlz** [comment]
Great stuff btw! Lots of detail. This is very much appreciated!!!

**Comment by Theodlz** [comment]
Hey guys, did you have any time to format all of that in a nice markdown file as used in skyportal's docs?

**Comment by celiune** [comment]
Hi @Theodlz, yes we did, please take a look at it when you have some time. 
[Additional steps infos to follow Skyportal setup .md](https://github.com/grandma-collaboration/skyportal_grandma_dev/files/10131875/Additional.steps.infos.to.follow.Skyportal.setup.md)
